pkg/controller/elasticsearch/certificates/remoteca/reconcile_test.go

```diff
@@ -101,14 +104,24 @@ func TestReconcile(t *testing.T) {
 						Data: map[string][]byte{certificates.CAFileName: []byte("cert2\n")},
 					},
 				},
+				ca: nil,
```

Comment:
I think we should pass in the `testCA` here and on L64 as well to make sure it does not get included when there are remote CA certs.

Reply:
Yes, that makes sense.